presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -65,7 +66,7 @@ private PlanFragmenter()
     {
     }
 
-    public static SubPlan createSubPlans(Session session, Metadata metadata, Plan plan)
+    public static SubPlan createSubPlans(Session session, Metadata metadata, SplitManager splitManager, Plan plan)
```

Comment:
Why? This change warrants an explanation of the motivation, especially since it's being added without anything using it.